### ISSUE TITLE
Unable to get geeklab theme running on Hugo.

### ISSUE THREAD

**Issue by MBZ-07** [thegeeklab/hugo-geekblog#18]
Hi, I'm in trouble to get geeklab theme running in hugo.

First, I created a new hugo site using `hugo new site mbz` command on Linux.
Then, I made a clone of geeklab theme in themes folder of my site with that command `git clone https://github.com/thegeeklab/hugo-geekblog.git`.
After that, I copied

```
baseURL = "http://localhost"
title = "Geekblog"
theme = "hugo-geekblog"

# Geekblog required configuration
pygmentsUseClasses = true
pygmentsCodeFences = true
disablePathToLower = true

# Needed for mermaid shortcodes
[markup]
  [markup.goldmark.renderer]
    unsafe = true
  [markup.tableOfContents]
    startLevel = 1
    endLevel = 9

[taxonomies]
  author = "authors"
  tag = "tags"

[mediaTypes]
  [mediaTypes."application/atom+xml"]
    suffixes = ["xml"]

[outputFormats]
  [outputFormats.Atom]
    name = "Atom"
    mediaType = "application/atom+xml"
    baseName = "atom"
    isPlainText = false
    rel = "alternate"
    isHTML = false
    noUgly = true
    permalinkable = false

[outputFormats.outputs]
  home = ["HTML", "ATOM"]
  page = ["HTML"]
  section = ["HTML"]
  taxonomy = ["HTML"]
  term = ["HTML", "ATOM"]

```  
  
to config.toml in my site as you mentioned here : https://hugo-geekblog.geekdocs.de/posts/getting-started/

so this is my website then

![Screenshot from 2020-12-04 21-49-45](https://user-images.githubusercontent.com/35532337/101214419-4bc55400-367c-11eb-805a-6c88716adebd.png)

and when I run "hugo server -D" command, I get that error !

![Screenshot from 2020-12-04 21-55-15](https://user-images.githubusercontent.com/35532337/101214452-5bdd3380-367c-11eb-9974-97c14a4597ab.png)

Please, could anyone help me, how to get that theme running without errors ? Thanks.

**Comment by xoxys** [comment]
Hi, please read the full getting started guide: https://hugo-geekblog.geekdocs.de/posts/getting-started/#option-2-clone-the-github-repository and/or the README of the theme

> If you want to use the theme from a cloned branch instead of a release tarball you’ll need to install gulp locally and run the default pipeline once to create all required assets.

tl;dr If you want to use this theme from the git source you need to install and run gulp to build required assets. As an alternative and recommended way use the prebuild tarball from the release page.

**Comment by MBZ-07** [comment]
Ok, I will follow the recommended way. I downloaded that file

![Screenshot from 2020-12-04 23-40-42](https://user-images.githubusercontent.com/35532337/101222644-79b19500-368a-11eb-9a32-7a35fb502fe0.png)


from here
![Screenshot from 2020-12-04 23-40-57](https://user-images.githubusercontent.com/35532337/101222638-75857780-368a-11eb-8931-b1d2fed4ac49.png)

What should I do next ?

**Comment by xoxys** [comment]
Extract the theme to the themes folder. Thats it.

**Comment by MBZ-07** [comment]
I got the website without neither posts nor sections above, how can I get them ?
![Screenshot from 2020-12-04 23-53-07](https://user-images.githubusercontent.com/35532337/101223310-06a91e00-368c-11eb-8111-5a45f2e143b3.png)

**Comment by xoxys** [comment]
I dont really unserstand the question. This is just a theme. You need to add the content on your own. Maybe you have to read first how hugo works in general.

**Comment by MBZ-07** [comment]
Just, I want to add the content of geeklab demo site (posts,nav bar) to my site to learn from them.

They are here when I cloned the repo
![Screenshot from 2020-12-05 00-02-45](https://user-images.githubusercontent.com/35532337/101223838-3efd2c00-368d-11eb-904e-5806befd9d79.png)
![Screenshot from 2020-12-05 00-03-15](https://user-images.githubusercontent.com/35532337/101223860-4ae8ee00-368d-11eb-9dab-32bd375347da.png)

**Comment by xoxys** [comment]
To get the demo site just copy the content folder and data folder from `exampleSite` to your own sites root folder.

**Comment by MBZ-07** [comment]
Yeah, I've done it, big thanks friend :+1:  , that's working .
